### PR TITLE
Grant steering committee members access to admin interface

### LINKED ISSUE #876: Steering Committee members aren't allowed into AMY
All access apart from AutoUpdateProfile is allowed only for admins, and people from Steering Committee have "steering committee" group which is not considered admin.

The workaround for now is that I'll add SC members to the "administrators" group.

### PR DESCRIPTION
**Pull request by chrismedrela**
Fix #876

### CODE REVIEW

File: workshops/models.py [hunk 1]
```diff
@@ -422,7 +422,8 @@ def is_admin(user):
         return False
     else:
         return (user.is_superuser or
-                user.groups.filter(name='administrators').exists())
+                user.groups.filter(name='administrators').exists() or
+                user.groups.filter(name='steering committee').exists())
```
Comment (pbanaszkiewicz) [inline workshops/models.py]: Make this a one query.

Remember that `is_admin` is run almost on every page, so if you increase number of queries here, AMY's performance may decrease.

**Comment by pbanaszkiewicz** [comment]
Fix needed, then I'll merge.

**Comment by chrismedrela** [comment]
@pbanaszkiewicz , done.

**Comment by pbanaszkiewicz** [comment]
@chrismedrela thanks!

**Pull request merged**